index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>My Portofolio</title>
+  </head>
+  <body>
+    <!--The main container: all elements are in-->
+    <main id="id-main-content">
+      <!--the document header-->
+      <header id="id-header-content">
+        <img src="/images/ic_menu.svg" alt="menu button" id="id-side-menu">
+      </header>
```

Comment:
The header tag should consist the logo with a link. Kindly make the necessary change.

Reply:
sorry, but there is no logo part on the template I choose.

Reply:
I thought the left image you introduced as a background image is a logo. In that case, it is ok. The next TSE may take this note so they won't need to react to this issue. :handshake: